scripts/tasks/check-license-headers.js

```diff
@@ -151,4 +150,9 @@ Please include the header or add an exception for the file in \`scripts/check-li
     }
 }
 
-check();
+// Check provided files, if any, otherwise check all files tracked by git
+check(
+    process.argv.length > 2
+        ? process.argv.slice(2)
+        : execSync('git ls-files', { encoding: 'utf-8' }).trim().split('\n')
```

Comment:
I really think this `git ls-files` is janky (why have this on top of an allowlist/denylist?), but I understand that this predates your PR.